rest-api/test.yaml

```diff
@@ -1,43 +1,45 @@
 # A combination of app.yaml and offline.yaml, with module: testing and version: 1,
-# to support running dev_appserver locally for tests in a way that can
+# to support running dev_appserver locally for tests in a way that can
 # run pipelines and our endpoints on one port.
 # This file should be kept in sync with app.yaml and offline.yaml.
-threadsafe: true
-runtime: python27
-api_version: 1
-
-builtins:
-- deferred: on
-
-includes:
-  - appengine-mapreduce/python/src/mapreduce/include.yaml
-
-module: testing
-version: 1
-
-handlers:
-- url: /_ah/pipeline.*
-  script: pipeline.handlers._APP
-  login: admin
-  secure: always
-- url: /_ah/queue/deferred
-  script: google.appengine.ext.deferred.deferred.application
-  login: admin
+threadsafe: true
+runtime: python27
+api_version: 1
+
+builtins:
+- deferred: on
```

Comment:
If you're not enabling appstats here (which seems fine?) you probably don't need the stats handler below.

Reply:
done